src/Cookie.php

```diff
@@ -40,7 +32,7 @@ public function __construct(string $name)
         $this->options = [];
     }
 
-    public function fromString(string $name): self {
+    public static function fromString(string $name): self {
```

Comment:
Make sense to keep code better.
In old case the initialization looks like:

```php
$cookie = new Cookie();
$fromStringCookie = $cookie->fromString('yourCookie');
```

Whit static option it looks like this:
```php
$cookie = Cookie::fromString('yourCookie');
```